### PR TITLE
Update docs for RN 0.60.0 and above

### PR DESCRIPTION
**Pull request by lb90**
Update docs with bits from #5572

### CODE REVIEW

**Comment by lb90** [comment]
Note: I can't test on MacOS and I don't know if the iOS instructions need update too

Also see: #5345, #5318

**Comment by alpha0010** [comment]
I think you can further simplify.  My class is:
```java
package com.yourproject;

import com.facebook.react.PackageList;
import com.facebook.react.ReactNativeHost;
import com.facebook.react.ReactPackage;
import com.reactnativenavigation.NavigationApplication;
import com.reactnativenavigation.react.NavigationReactNativeHost;

import java.util.List;

public class MainApplication extends NavigationApplication {

    @Override
    protected ReactNativeHost createReactNativeHost() {
        return new NavigationReactNativeHost(this) {
            @Override
            protected String getJSMainModuleName() {
                return "index";
            }
        };
    }

    @Override
    public boolean isDebug() {
        return BuildConfig.DEBUG;
    }

    @Override
    public List<ReactPackage> createAdditionalReactPackages() {
        @SuppressWarnings("UnnecessaryLocalVariable")
        List<ReactPackage> packages = new PackageList(this).getPackages();
        // Add packages here that cannot yet be autolinked.
        return packages;
    }
}
```

**Comment by lb90** [comment]
Thanks! I don't have much knowledge on that, but I'm going to update this with your code.

**Comment by lb90** [comment]
Ready for review

File: docs/docs/Installing.md [hunk 1]
```diff
@@ -369,6 +371,54 @@ import java.util.List;
...
+
+-import android.app.Application;
+-import android.content.Context;
+ import com.facebook.react.PackageList;
+-import com.facebook.react.ReactApplication;
+ import com.facebook.react.ReactNativeHost;
+ import com.facebook.react.ReactPackage;
+-import com.facebook.soloader.SoLoader;
+-import java.lang.reflect.InvocationTargetException;
+-import java.util.List;
```
Comment (zeevl) [inline docs/docs/Installing.md]: this import needs to stay

**Pull request closed without merging**